docs/install/ipfs-companion.md

```diff
@@ -1,9 +1,72 @@
 ---
 title: IPFS companion
-description: Learn what the IPFS Companion browser extention is, why it's useful, and how you can use it!
-issueUrl: https://github.com/ipfs/ipfs-docs/issues/413
+description: "The IPFS companion allows you to interact with your IPFS node and the extended IPFS network through your browser! The add-on is available for Brave, Chrome, Edge, Firefox, and Opera! It enables support for `ipfs://` addresses, automatically loads websites and file paths from an IPFS gateway, allows you to easily import and share a file with IPFS, and a bunch of other cool stuff!"
 ---
 
 # IPFS companion
 
-<ContentStatus />
+The IPFS companion allows you to interact with your IPFS node and the extended IPFS network through your browser! The add-on is available for Brave, Chrome, Edge, Firefox, and Opera! It enables support for `ipfs://` addresses, automatically loads websites and file paths from an IPFS gateway, allows you to easily import and share a file with IPFS, and a bunch of other cool stuff!
```

Comment:
There's a few instances of "the IPFS companion" in here -- could you please use "IPFS Companion", no "the", capitalized, instead?